### ISSUE TITLE
Adding hint does not work

### ISSUE THREAD

**Issue by tieorange** [julienandco/google_places_autocomplete_text_field#4]
Code:
```
GooglePlacesAutoCompleteTextFormField(
            textEditingController: controller,
            googleAPIKey: AppConstants.googlePlacesApiKey,
            debounceTime: 550,
            decoration: const InputDecoration(
              enabledBorder: OutlineInputBorder(
                borderSide: BorderSide(color: Color(0xff0E9447), width: 2.0),
              ),
              focusedBorder: OutlineInputBorder(
                borderSide: BorderSide(color: Color(0xff0E9447), width: 2.0),
              ),
              hintText: 'AppStrings.email',
              labelText: 'AppStrings.email',
              alignLabelWithHint: true,
              hintStyle: TextStyle(color: Colors.red),
            ),
            maxLines: 1,
            overlayContainer: (child) => Material(
              elevation: 1,
              borderRadius: BorderRadius.circular(12),
              child: child,
            ),
            getPlaceDetailWithLatLng: (prediction) {
              log('placeDetails${prediction.toJson()}');
              // prediction.structuredFormatting.mainText
            },
            itmClick: (prediction) {
              final description = prediction.description ?? '';

              controller
                ..text = description
                ..selection = TextSelection.fromPosition(
                  TextPosition(offset: description.length),
                );
            },
          ),
          ```

Result:
<img width="395" alt="image" src="https://github.com/julienandco/google_places_autocomplete_text_field/assets/3984517/41fd27fb-68be-4129-84aa-5798ef5d5e01">

**Comment by julienandco** [comment]
@tieorange Thank you for submitting this issue! I found the problem, it is due to a duplicate field that I will remove with the next version. For now, use `inputDecoration` instead of `decoration`.

**Comment by julienandco** [comment]
@tieorange The issue is fixed in the version `0.1.3`
